worker/vcs/manager.go

```diff
@@ -164,6 +165,25 @@ func (vm *Manager) CloneVersionRepository(event *api.Event) error {
 		steplog.InsertStepLog(event, steplog.CloneRepository, steplog.Stop, err)
 		return fmt.Errorf("Unable to clone repository for version: %v\n", err)
 	}
+
+	err = vm.ensureCommitValid(event, destPath, worker)
+	if err != nil {
+		return err
+	}
+
+	vm.formatVersionName(event)
```

Comment:
CloneVersionRepository() just cares about cloning repo, does not need to care about version name. So this logic should not be put here. Please refer to [worker/worker.go#L348-L380](https://github.com/caicloud/cyclone/blob/v1-preview/worker/worker.go#L348-L380).